src/main/java/com/pusher/client/PusherOptions.java

```diff
@@ -43,39 +43,56 @@
     private int maxReconnectGapInSeconds = MAX_RECONNECT_GAP_IN_SECONDS;
 
     /**
-     * @deprecated
-     * Please use isForceTLS
+     * @deprecated 2.2.0
+     * Please use isUseTLS
      */
     @Deprecated
     public boolean isEncrypted() {
-        return forceTLS;
+        return useTLS;
     }
 
     /**
-     * @deprecated
-     * Please use setForceTLS
+     * @deprecated 2.2.0
+     * Please use setUseTLS
      */
     @Deprecated
     public PusherOptions setEncrypted(final boolean encrypted) {
-        this.forceTLS = encrypted;
+        this.useTLS = encrypted;
+        return this;
+    }
+
+    /**
+     * @deprecated 2.3.0
```

Comment:
I think that given we realised the mistake in a matter of hours, we can get away with calling this a patch release. I would be incredibly surprised if anyone has started building against this version, never mind released with it